resources/translations/en.edn

```diff
@@ -83,16 +83,23 @@
                    :catalogue-item "Catalogue item"
                    :catalogue-items "Catalogue items"
                    :categories "Categories"
+                   :category "Category"
+                   :category-title "Title"
+                   :category-description "Description"
+                   :category-children "Subcategories"
                    :change "Change"
                    :comment "Comment"
                    :copy-as-new "Copy as new"
                    :create-catalogue-item "Create catalogue item"
+                   :create-category "Create category"
                    :create-form "Create form"
                    :create-license "Create license"
                    :create-organization "Create organization"
                    :create-resource "Create resource"
                    :create-workflow "Create workflow"
                    :created "Created"
+                   :delete "Delete"
+                   :delete-category "Delete category?"
```

Comment:
We could perhaps make do with more generic "Really delete?" version? Not having to potentially have so many similar translations.

Reply:
replaced this with more generic translation. not sure if the swedish version is grammatically on point though..